### PR TITLE
item correction

### PR DESCRIPTION
**Pull request by eppc0330**
* **Addressed Issue(s)**: 

* **Server Mode**: Renewal

* **Description of Pull Request**: 

Some item corrections

### CODE REVIEW

File: db/re/item_db_etc.yml [hunk 1]
```diff
@@ -21557,6 +21583,36 @@ Body:
     Type: Etc
     Buy: 10
     Weight: 100
+  - Id: 6720
+    AegisName: Status_Reset_Coupon
+    Name: Status Reset Coupon
+    Type: Cash
```
Comment (Atemo) [inline db/re/item_db_etc.yml:21589]: This item is currently type Cash, why putting it in `item_db_etc.yml` ?

File: db/re/item_db_usable.yml [hunk 2]
```diff
@@ -11011,7 +10944,7 @@ Body:
       NoAuction: true
     Script: |
       specialeffect2 EF_HEAL3;
-      bonus_script "{ bonus2 bHPRegenRate,(MaxHp/100*6),3000; }",600;
+      sc_start4 SC_G_LIFEPOTION,600000,-6,3,0,0;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:10947]: Do you know if the effect stacks with `SC_L_LIFEPOTION` `SC_S_LIFEPOTION` `SC_M_LIFEPOTION` on KRO?

**Review by Atemo — COMMENTED** [review]
Look good!

File: db/re/item_db_etc.yml (hunk 1 above)
Comment (eppc0330) [reply db/re/item_db_etc.yml:21589]: Sry it is etc, not cash. I forgot to change it

**Review by Atemo — APPROVED** [review]

**Pull request merged**